### DISCUSSION TITLE
C.V. using quatro?

### DISCUSSION THREAD

**Discussion by dmbates** [quarto-dev/quarto-cli#705] [Q&A]
I get some hits when searching for `C.V. using R Markdown` but I don't see anything yet for quatro.  Has anyone tried to produce a C.V. using quatro?  I have a bib file with my publications including a DOI reference for most of them and would like to create the vita from those.

Perhaps quatro is overthinking things and maybe I just need a way of producing a publication listing from a bibtex file.

**Comment by cderv** [comment]
FWIW **rmarkdown** currently have some tools and template to create CV for R users: 

* https://github.com/rstudio/pagedown#resume-pagedownhtml_resume
* https://github.com/mitchelloharawild/vitae
* https://github.com/nstrayer/datadrivencv

You may have found those. 

Once custom format are possible within Quarto, I plan to try porting at least the fist one (`pagedown::html_resume()`) to be used within Quarto to be available cross-language.

I'll post update on that if no one does it before.

**Reply by cderv** [reply]
See also https://github.com/quarto-dev/quarto-cli/discussions/2431 about the need to support reference by section

**Comment by barizona** [comment]
I still found no out of the box solution in Quarto to create a CV (html and pdf). Any plans for this?

**Reply by mcanouil** [reply]
Plans yes, at least far as I know, but no timeline.

To note, this relies mostly on creating a new format/template using Paged.js.

**Reply by jeremyforest** [reply]
would be interested but not too sure where to start on this. Any suggestions ?

**Reply by cderv** [reply]
With Quarto 1.4 new support for Typst (https://quarto.org/docs/prerelease/1.4/typst.html), it think it would be a good starting point to adapt a CV format from Typst for Quarto as an extension. Example of Typst format adapted for use with Quarto (https://github.com/quarto-ext/typst-templates). We don't have one for CV yet but I seem there are some tries already in Typst

* https://mattrighetti.com/2023/10/25/i-rewrote-my-cv-in-typst
* https://github.com/skyzh/chicv
* https://github.com/bamboovir/typst-resume-template

And probably others 

I added this to follow up
* https://github.com/quarto-ext/typst-templates/issues/6
